crates/sui-graphql-rpc/src/types/object.rs

```diff
@@ -764,24 +753,14 @@ impl Object {
         serialized: Option<Vec<u8>>,
```

Comment:
Do we ever pass `None` to this now?

Reply:
Yes I think this can be non optional now, we'll always have bcs